apps/portal/src/pages/index.astro

```diff
@@ -1,4 +1,6 @@
 ---
+import Anchor_ from '../../../css-workshop/src/components/Anchor.astro';
```

Comment:
undo this one, the `portal` is a completely different codebase from `css-workshop`

Reply:
Reverted.